package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "resin-typescript-skeleton",
+    "version": "0.0.1",
+    "description": "Skeleton template for a Resin TypeScript project",
+    "homepage": "https://github.com/resin-io/resin-typescript-skeleton#readme",
+    "types": "build/index.d.ts",
```

Comment:
I think that this should be `"types": "src/index.ts"`. It's what myself and @Page- have been using and it makes the most sense IMO. Firstly `.d.ts` files do not need to be generated (assuming `index.d.ts` wasn't handcrafted) and the types header in `package.json` is only useful when using typescript anyway, implying there is a typescript compiler available. In addition it feels much cleaner. A good example of this is https://github.com/resin-io/resin-docker-build and https://github.com/resin-io-modules/resin-bundle-resolve and these two modules have been consumed happily from JS and TS with minor setup.